### PR TITLE
Support multiarch

### PR DESCRIPTION
**Pull request by agonbar**
Ideally we could support all native golang architectures, its just a matter of creating the respective Dockerfiles. I would omit any OS that is not Linux as we depend too much on iptables right now. Maybe on the future removing layers would simplify the maintenance.

This solves #103

### CODE REVIEW

**Comment by jack1902** [comment]
@agonbar i have rebased this PR against master

**Comment by jack1902** [comment]
@agonbar i can see that we can add `tags: ['latest']` to the manifest file for dockerhub but i don't understand how we can set it for versions like `v1.1` , `v1.2` automatically? prehaps we just have to do something manually when pushing out versions for multiple architectures?

File: multi-arch-manifest.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
-image: subspacecommunity/subspace
+image: subspacecommunity/subspace:latest
+tags: ['1.1.1', '1.1', '1']
```
Comment (jack1902) [inline multi-arch-manifest.yaml:2]: So when we update to `2` we then have to change this to `[2.CURRENT_MINOR_VERSION.CURRENT_BUG_FIX_VERSION, 2.CURRENT_MINOR_VERSION, 2]` ?

I assume that we won't ever build off of master in dockerhub as we only want it to trigger when we release a version?

Comment (agonbar) [reply multi-arch-manifest.yaml:2]: Hmmmm true, we can avoid pushing incomplete versions as images in the master branch

**Comment by agonbar** [comment]
Also, if we merge this, shoud #43 be closed? At least for now.

**Pull request merged**